media-capabilities/decodingInfo-webrtc.any.js

```diff
@@ -225,3 +372,79 @@ validVideoCodecs.forEach(codec => {
 }, "Test that decodingInfo returns supported true for the codec " + codec + (isWorkerEnvironment ? "" : " returned by RTCRtpReceiver.getCapabilities()"))}
 );
 
+promise_test(t => {
+  return promise_rejects_js(t, TypeError, navigator.mediaCapabilities.decodingInfo({
+    type: 'webrtc',
+    audio: { contentType: 'audio/webm; codecs="opus"; foo="bar"' },
+  }));
+}, "Test that decodingInfo rejects if the audio configuration contentType has more than one parameters");
+
+promise_test(t => {
+  return promise_rejects_js(t, TypeError, navigator.mediaCapabilities.decodingInfo({
+    type: 'webrtc',
+    audio: { contentType: 'audio/webm; foo="bar"' },
+  }));
+      }, "Test that decodingInfo rejects if the audio configuration contentType has one parameter that isn't codecs");
+
+promise_test(t => {
+  // VP9 has a default color space of BT.709 in the codec string. So this will
+  // mismatch against the provided colorGamut and transferFunction.
+  let bt709Config = videoConfigurationWithDynamicRange;
+  bt709Config.contentType = 'video/webm; codecs="vp09.00.10.08"';
+  return navigator.mediaCapabilities
+      .decodingInfo({
+        type: 'webrtc',
+        video: bt709Config,
+      })
+      .then(ability => {
+        assert_equals(typeof ability.supported, 'boolean');
+        assert_equals(typeof ability.smooth, 'boolean');
+        assert_equals(typeof ability.powerEfficient, 'boolean');
+        assert_equals(typeof ability.keySystemAccess, 'object');
+        assert_false(ability.supported);
+      });
+}, 'Test that decodingInfo with mismatched codec color space is unsupported');
+
+promise_test(t => {
+  return promise_rejects_js(t, TypeError, navigator.mediaCapabilities.decodingInfo({
+    type: 'webrtc',
+    video: {
+      contentType: 'video/webm; codecs="vp09.00.10.08"',
+      width: 800,
+      height: 600,
+      bitrate: 3000,
+      framerate: 24,
+      hdrMetadataType: ""
```

Comment:
Instead of an empty string, could there be an actual value? Main goal for me is to test that the presence of `hdrMetadataType` rejects when type is `webrtc`.

```suggestion
      hdrMetadataType: "smpteSt2086"
```

Reply:
This test was removed.